.drone.jsonnet

```diff
@@ -136,6 +136,9 @@ local setup_ci = {
     'make ./_out/sonobuoy',
     'make ./_out/kubectl',
   ],
+  environment: {
+    "BUILDKIT_FLAVOR": "cross",
```

Comment:
I couldn't find docs on this. Have a ref?

Reply:
it's a feature of our builld-container: https://github.com/talos-systems/build-container/pull/46